sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -72,26 +80,19 @@ dependencies {
   compile library.java.proto_google_cloud_pubsublite_v1
   compile library.java.proto_google_common_protos
   compile library.java.protobuf_java
+  compile library.java.protobuf_java_util
   compile library.java.slf4j_api
+  compile library.java.vendored_grpc_1_26_0
+  compile library.java.vendored_guava_26_0_jre
+
   compile "com.google.api.grpc:proto-google-cloud-bigquerystorage-v1:0.90.0"
   compile "com.google.api.grpc:proto-google-cloud-bigtable-admin-v2:1.14.0"
   compile "com.google.api.grpc:proto-google-cloud-spanner-v1:1.59.0"
-  compile "com.google.api:api-common:1.8.1"
-  compile "com.google.apis:google-api-services-storage:v1-rev20200611-1.30.10"
-  compile "com.google.code.gson:gson:2.8.6"
-  compile "com.google.errorprone:error_prone_annotations:2.3.3"
-  compile "com.google.protobuf:protobuf-java-util:3.11.1"
-  compile "io.grpc:grpc-api:1.27.2"
-  compile "org.apache.beam:beam-vendor-guava-26_0-jre:0.1"
-  compile "org.apache.httpcomponents:httpclient:4.5.12"
-  compile "org.apache.httpcomponents:httpcore:4.4.13"
-  compile "org.checkerframework:checker-qual:2.0.0"
-  compile library.java.jackson_core
-  compile "org.hamcrest:hamcrest:2.1"
   compile "org.threeten:threetenbp:1.4.4"
-  compile "org.apache.beam:beam-vendor-grpc-1_26_0:0.3"
+
```

Comment:
these new lines are just to separate lines where library.java variables are used vs hardcoded once and to separate compile lines from testCompile lines

Reply:
Nit: there are already hardcoded declarations at the top of the `dependencies` block.